Dockerfile-freeradius

```diff
@@ -3,7 +3,7 @@
 #
 # Build image:
 # 1. git pull git@github.com:lirantal/daloradius.git
-# 2. docker build . -t lirantal/daloradius
+# 2. docker build . -t lirantal/daloradius -f Dockerfile-freeradius
```

Comment:
the `.` already specifies the dockerfile already, no? isn't that making the `-f` at the end extranous? I assume it is one or the other.

Reply:
The . would build the Dockerfile and not the Dockerfile-freeradius. That is why I added it.

Reply:
What I meant is that I think the correct syntax is this, no?

```
docker build -t lirantal/daloradius -f Dockerfile-freeradius
```

Reply:
Ah, missed the . 
You are correct.